.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude = docs, scanpy/tests
```

Comment:
Why exclude docs and tests?

I expect some pathological things in tests (unused imports for pytest fixtures), can we disable specific checks by path?

Reply:
Oh, I did this before to get at least some sanity. There was so much stuff to fix that I excluded them and then forgot.

https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores